recipe_templates/sunpy/meta.yaml

```diff
@@ -4,13 +4,13 @@ package:
 
 source:
   fn: sunpy-{{version}}.tar.gz
-  url: https://pypi.io/packages/source/s/sunpy/sunpy-{{ version }}.tar.gz 
+  url: https://pypi.python.org/packages/75/d3/9b5578d40feaa7adb535b4ba40a455404989482f4fca1b2bf18e4bdb3ca9/sunpy-{{ version }}.tar.gz
   md5: {{ md5 }}
 
 build:
   number: 0
   script: python setup.py install --offline --no-git --single-version-externally-managed --record record.txt
-  skip: True  # [py>27]
+  skip: True  # [win and py35]
```

Comment:
could be...I have too many packages I'm keeping track of right now.  Just looked, and you are right, it is *building* sunpy successfully on Windows/python 3.4, but it fails when it tries to test the build. Not sure why, let's see what happens on the branch build